packages/core/src/extensions/renderer-api/tokens.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+export * from "../../renderer/components/cluster-settings/cluster-settings-menu-injection-token";
```

Comment:
The idea is not to export these to the legacy extension API, but instead have a new package which just contain these tokens.

That way a new feature extension would just import the token package (declaring it as a `peerDependency`)

Reply:
Removed legacy exports.